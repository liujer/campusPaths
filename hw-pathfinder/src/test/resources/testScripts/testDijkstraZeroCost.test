# Tests if path from a node to itself is zero
# using Dijkstra's algorithm
CreateGraph A
AddNode A one

FindPath A one one