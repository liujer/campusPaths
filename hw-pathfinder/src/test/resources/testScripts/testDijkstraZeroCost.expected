# Tests if path from a node to itself is zero
# using Dijkstra's algorithm
created graph A
added node one to A

path from one to one:
total cost: 0.000
